code/game/objects/items/weapons/storage/judgement_sheath.dm

```diff
@@ -0,0 +1,176 @@
+/obj/item/storage/belt/sheath/judgement
+	icon_state = "sheath_judgement"
+	name = "curved sheath"
+	desc = "Made to store curved swords. It is dark blue. It has a patterned gold head on it's bottom part. On inner side of it's mouth you can notice strange cyan crystal."
+	can_hold = list(
+		/obj/item/tool/sword/saber,
+		/obj/item/tool/sword/katana
```

Comment:
Might want to remove tha katana. It's no way curved.

Reply:
It's curved, as sprite same as saber